.gitignore

```diff
@@ -3,5 +3,7 @@
 .packages
 .pub/
 build/
+.idea
```

Comment:
We explicitly don't ignore IDE files. Please put this in your `.gitignore_global`